articles/enroll-hosts.md

```diff
@@ -27,6 +27,8 @@ The `--type` flag is used to specify the fleetd installer type.
 
 A `--fleet-url` (Fleet instance URL) and `--enroll-secret` (Fleet enrollment secret) must be specified in order to communicate with Fleet instance.
 
+> `fleetctl` on macOS/Linux requires `umask` to be `022`/`002` and `/tmp` (used during package generation) has to be mounted without `noexec`.
```

Comment:
- `umask` to be `022`/`002`
- `/tmp` (used during package generation) has to be mounted without `noexec`

@lucasmrod are these defaults for macOS/Linux? If yes, I think we can remove these from the docs. I think it makes using fleetctl seem harder than it needs to be.

If not, what action does the user have to take to make sure they have these?

Reply:
> are these defaults for macOS/Linux?

Probably the default for macOS.
Linux is a big world, but seems to be the case for most distros.

> If not, what action does the user have to take to make sure they have these?

- Set `umask` to the supported values by running `umask 022`.
- Make sure where /tmp was mounted does not have `noexec`.

--

Do we want a PR to revert this change?

Reply:
Hmm, I think so. If we're confident it's the default I think most users won't need to know.

I opened a PR here: https://github.com/fleetdm/fleet/pull/23151

That said, please keep up the guide edits!

Reply:
Sounds good.

Feel free to add a comment on https://github.com/fleetdm/fleet/issues/22877.